### ISSUE TITLE
Allow unsafe HTTPS to InfluxDB

### ISSUE THREAD

**Issue by ptrcarta** [timescale/outflux#53]
InfluxDB documentation on [https setup](https://docs.influxdata.com/influxdb/v1.7/administration/https_setup/) suggests setting up HTTPS with a self-signed certificate.
Outflux doesn't allow self signed certificates, but allowing them would be easily done by adding an option flag --insecureSSL and `InsecureSkipVerify: insecure` to
https://github.com/timescale/outflux/blob/34dbb29fb01bf320f357efeb13c33a8398164646/internal/connections/influx_connection.go#L54

### workaround:
If you have a working go toolchain, you can work around this limitation by fetching and modifying the code

```sh
go get github.com/timescale/outflux
cd $GOPATH/src/github.com/timescale/outflux
nano internal/connections/influx_connection.go
```
replacing line 54 (HTTPConfig creation) with the following
```go
clientConfig := influx.HTTPConfig{Addr: params.Server, Username: user, Password: pass, InsecureSkipVerify: true}
```
and then finally compiling and installing outflux with:
```sh
dep ensure -v
cd cmd/outlux
go install 
```
The binary program will be in $GOPATH/bin

**Comment by dianasaur323** [comment]
@blagojts depending on how long it takes to add this option, I think we can consider adding this in? @ptrcarta we would also certainly welcome contributions :)

**Comment by ptrcarta** [comment]
thanks for the response. I will gladly submit a pull request in the next days.

**Comment by blagojts** [comment]
We merged this as part of #58 to master and made a new release with the binaries (0.1.2).
